### DISCUSSION TITLE
2023/gpt-sw3/

### DISCUSSION THREAD

**Discussion by giscus** [mirontoli/chuvash-eu-comments#1] [General]
# 2023/gpt-sw3/

AI Sweden har publicerat den första svenska språkmodellen - GPT-SW3:
Nyheten på Computer Sweden Samlingssida på AI Sweden Språkmodeller från AI Sweden på HuggingFace I det här inlägget vill jag beskriva hur man kommer igång om man vill testa det. Min setup är följande:
En MacBook Pro (men det kan vara vilken dator som helst egentligen) Python 3.8^ + venv (om det lite senare) Visual Studio Code Jupyter Notebook Extension till Visual Studio Code Datoruppsättning Om inte du har gjort det tidigare, installera Visual Studio Code

http://localhost:1313/2023/gpt-sw3/

**Comment by mirontoli** [comment]
Test1
